File: JUNGOL/Beginner/#1009 각 자리수의 역과 합/각 자리수의 역과 합.cpp
#include <iostream>

using namespace std;

int main() {
	while(1){
		int n; cin>>n;
		if(n == 0) break;

		int sum=0, reverse=0;
		while(n){
			reverse = reverse*10+(n%10);
			sum += (n%10);
			n /= 10;
		}
		
		printf("%d %d\n", reverse, sum);
	}
}

File: JUNGOL/Beginner/#1009 각 자리수의 역과 합/각 자리수의 역과 합.py
def main():
    while True:
        n = int(input())
        if n == 0: break

        sum,reverse = 0,0
        while n > 0:
            reverse = reverse*10 + n%10
            sum += (n%10)
            n //= 10

        print(reverse, sum, sep=' ')

main()
